### ISSUE TITLE
Questions about ADA

### ISSUE THREAD

**Issue by hot-dog** [NVlabs/stylegan2-ada-pytorch#292]
As the paper state, StyleGAN2 use non-saturating loss and overfitting heuristics rt is defined as rt=E[sign(Dtrain)]. My question is if i switch to a different GAN, such as LSGAN or Vanilla GAN, whose loss is not non-saturating loss, can I still use the rt defined above? Do I need to make appropriate modifications to rt?
